obsreport/obsreport_scraper.go

```diff
@@ -46,19 +65,72 @@ type ScraperSettings struct {
 }
 
 // Deprecated: [v0.64.0] use MustNewScraper.
-var NewScraper = MustNewScraper
+func NewScraper(cfg ScraperSettings) *Scraper {
+	scr, err := newScraper(cfg, featuregate.GetRegistry())
+	if err != nil && cfg.ReceiverCreateSettings.Logger != nil {
+		cfg.ReceiverCreateSettings.Logger.Warn("Error creating an obsreport.Scraper", zap.Error(err))
+	}
+	return scr
+}
```

Comment:
Already changed, https://github.com/open-telemetry/opentelemetry-collector/blob/main/obsreport/obsreport_scraper.go#L49, you need to rebase this.

Reply:
Thanks for the review! Rebased so things should be up to date now.